src/main/java/io/vertx/core/spi/metrics/DatagramSocketMetrics.java

```diff
@@ -43,8 +43,9 @@
    * Called when a socket is listening. For example, this is called when an http or net server
    * has been created and is listening on a specific host/port.
    *
+   * @param localName
    * @param localAddress the local address the net socket is listening on.
    */
-  void listening(SocketAddress localAddress);
+  void listening(String localName, SocketAddress localAddress);
```

Comment:
maybe we should have in the doc a difference between API and SPI ? so it is clear that SPI changes won't affect normal users.